addOns/websocket/src/main/zapHomeFiles/scripts/templates/websocketpassive/Base64 Disclosure.js

```diff
@@ -37,19 +39,31 @@ function scan(helper,msg) {
                 print("Decoded Evidence: " + decodedEvidence);
             }
 
-            helper.newAlert()
-                .setRiskConfidence(RISK_INFO, CONFIDENCE_MEDIUM)
-                .setName("Base64 Disclosure in WebSocket message (script)")
-                .setDescription("A Base64-encoded string has been found in the websocket incoming message. Base64-encoded data may contain sensitive " +
-                                "information such as usernames, passwords or cookies which should be further inspected. Decoded evidence: "
-                                + decodedEvidence + ".")
-                .setSolution("Base64-encoding should not be used to store or send sensitive information.")
-                .setEvidence(evidence)
-                .raise();
+            raiseAlert(helper, evidence, decodedEvidence);
         });
     }
 }
 
+function raiseAlert(helper, evidence, decodedEvidence){
+    createAlertBuilder(helper, evidence, decodedEvidence).raise();
+}
+
+function createAlertBuilder(helper, evidence, decodedEvidence){
+    return helper.newAlert()
+        .setPluginId(getId())
+	    .setRiskConfidence(RISK_INFO, CONFIDENCE_MEDIUM)
+	    .setName("Base64 Disclosure in WebSocket message")
+	    .setDescription("A Base64-encoded string has been found in the websocket incoming message. Base64-encoded data may contain sensitive " +
+	                    "information such as usernames, passwords or cookies which should be further inspected. Decoded evidence: "
+	                    + decodedEvidence + ".")
+	    .setSolution("Base64-encoding should not be used to store or send sensitive information.")
+	    .setEvidence(evidence);
+}
+
+function getExampleAlerts(){
+    return [createAlertBuilder(WebSocketPassiveScript.getExampleHelper(), "example", "example").build().getAlert()];
```

Comment:
In other scripts the evidence was not added, remove here or add in other scripts?

Reply:
Thats because the evidence field isnt usually included in the webpage .. but the description is, and in this case those strings end up in the description :)